components/navbar/Navbar.js

```diff
@@ -91,10 +91,6 @@ export default function Navbar() {
       name: "Docs",
       url: "/docs",
     },
-    {
```

Comment:
If you are removing playground from the nav bar, you need to remove [this test](https://github.com/EddieHubCommunity/LinkFree/blob/main/tests/playground.spec.js#L11-L15)

Reply:
oh good spot, thank you 👍